commit a13995fbc7e4548dda7b603bbb117c5fd3d2a936 (refs/changes/94/3094/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2017-12-11T17:55:43-08:00 (1 year, 10 months ago)
    
    RICHMOND-138 mfg allocation table columns are incorrect
    Reviewed by: Dale Ghent <dale.ghent@joyent.com>
    Approved by: Cody Mello <melloc@joyent.com>

diff --git a/allocations.md b/allocations.md
index d1b38af..12fecd9 100644
--- a/allocations.md
+++ b/allocations.md
@@ -15,8 +15,8 @@ which are used as part of SMBIOS and other product naming.
 
 ## Products, SKU, Common Name
 
-| SKU | Product ID | Common Name |
-|-----|------------|-------------|
+| SKU | Common Name | Product ID |
+|-----|-------------|------------|
 | Richmond A | JCP 1101 | 600-0002 |
 | Richmond B | JCP 1201 | 600-0004 |
 | Richmond C | JCP 1102 | 600-0006 |
